packages/engine/src/worker/runner/python/run.sh

```diff
@@ -7,11 +7,15 @@ VENV_ACTIVATE_PATH="$SCRIPT_DIR/runner_venv/bin/activate"
 
 . $VENV_ACTIVATE_PATH
 
+# Find the latest "hash_engine_lib" compiled shared library
+LATEST_LIBRARY=$(find "$SCRIPT_DIR/../../../../target" -name "libhash_engine_lib.so" -print0 | xargs -r -0 ls -1 -t | head -1)
```

Comment:
I'm a bit too tired to look up the arguments for these utilities (sorry 😞 ) but does this recursively search or is it depth limited? Just want to make sure it works for cross-compilation
<img width="261" alt="image" src="https://user-images.githubusercontent.com/25749103/159576433-d1b0413f-d471-4e82-b239-39147399f266.png">

Reply:
I'd leave that for you to test with your aarch64 setup 🙂 
It should work for cross compiling. Previously we had this hardcoded but this doesn't work with bindeps-artifacts. Also, previously, when we compiled a release build, the debug build will never be picked up again until the release build is `clean`ed.

Reply:
Seems to be working locally